google/cloud/pubsub/internal/batching_publisher_tracing_connection.cc

```diff
@@ -33,8 +33,9 @@ class BatchingPublisherTracingConnection : public pubsub::PublisherConnection {
   ~BatchingPublisherTracingConnection() override = default;
 
   future<StatusOr<std::string>> Publish(PublishParams p) override {
-    auto span =
-        internal::MakeSpan("pubsub::BatchingPublisherConnection::Publish");
+    auto span = internal::MakeSpan("publish scheduler");
+    span->SetAttribute("cloud-cxx.function",
```

Comment:
I just remembered that there is `code.function`:

https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/span-general/#source-code-attributes

Maybe that is a better match?

Reply:
How does this change relate to #12782?  Shouldn't we adopt the semantic conventions for function name here too?

Reply:
Whoops. I thought I got did this one too, I changed the test.cc in my commit but missed the .cc (file didn't save and I only pushed the changes in the test.cc). 

Fixed.